constrained-voucher.mkd

```diff
@@ -193,14 +195,14 @@ They are presented in the order "voucher-request", followed by a "voucher" respo
 the order that they occur in the protocol.
 
 The constrained voucher request MUST be signed by the Pledge.
-It signs using the private key associated with its IDevID X.509 certificate, or if an IDevID is not available, then the private key associated with its manufacturer-installed raw public key (RPK).
+It signs using the private key associated with its IDevID PKIX certificate, or if an IDevID is not available, then the private key associated with its manufacturer-installed raw public key (RPK).
```

Comment:
```suggestion
It signs using the private key associated with its IDevID certificate, or if an IDevID is not available, then the private key associated with its manufacturer-installed raw public key (RPK).
```

Reply:
Pledge has an IDevID certificate, not a PKIX certificate nor a IDevID PKIX certificate.